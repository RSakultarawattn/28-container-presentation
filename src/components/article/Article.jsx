import React from 'react';
import PropTypes from 'prop-types';

export default function Article({ article }) {
  return (
    <div>
      <h1>{article.title}</h1>
      <p>{article.author}</p>
      <p>{article.content}</p>
    </div>
  );
}
Article.propTypes = {
  article: PropTypes.object,
};
