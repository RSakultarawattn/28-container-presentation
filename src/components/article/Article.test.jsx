import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Article from './Article';

describe('article item component', () => {
  afterEach(() => cleanup());
  it('renders one single article', () => {
    const { asFragment } = render(<Article 
    
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
