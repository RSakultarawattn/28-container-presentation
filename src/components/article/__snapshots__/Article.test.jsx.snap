// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`article item component renders one single article 1`] = `
<DocumentFragment>
  <a
    class="Article"
  >
    <figure>
      <img />
      <figcaption />
    </figure>
  </a>
</DocumentFragment>
`;
